.packit.yaml

```diff
@@ -20,6 +20,8 @@ jobs:
     targets:
       - fedora-42-x86_64
       - epel-9-aarch64
+      - epel-10-x86_64
+      - centos-stream-10-x86_64
```

Comment:
is this missing `centos-stream-10-aarch64`

Reply:
@avishayt asked for centos-stream. It doesn't take time to build them, so why not to have?